jbehave-support-core/src/main/java/org/jbehavesupport/core/ssh/SshTemplate.java

```diff
@@ -43,12 +46,17 @@
     }
 
     public SshTemplate(SshSetting sshSetting, String timestampFormat, RollingLogResolver rollingLogResolver) {
```

Comment:
Do we want to deprecate or not?

Reply:
It's good idea to deprecate this one. We can talk about it on call :)

Reply:
It's deprecated now.